tests/torch_compile/test_compile.py

```diff
@@ -14,22 +14,11 @@
 
 torch._dynamo.config.cache_size_limit = 100
 
-import collections
-
 
 def get_dynamo_stats():
     # TODO: consider deepcopy'ing the entire counters struct and
     # adding a helper to do subtraction on it
```

Comment:
Is this comment still valid?

Reply:
Thanks for the comments, I think they should be removed.  Will update.